docker-compose.yml

```diff
@@ -1,6 +1,17 @@
 mysql-1:
   image: percona:5.7
-  command: --server-id=1 --log-bin=mysql-bin --binlog-format=ROW --sync-binlog=1 --log-slave-updates=ON --gtid-mode=ON --enforce-gtid-consistency=ON --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci --max-connections=1000 --read-only=OFF
+  command: --server-id=1
+    --log-bin=mysql-bin
+    --max-binlog-size=4096
```

Comment:
Would this change cause other problems in the tests? Maybe we can no longer replicate certain conditions if the binlog size is too small? Can we dynamically set this variable?

I don't really have an objection but I'm just speculating.

Reply:
Well, it shouldn't, otherwise we have a bug, no?

This actually is uncovering a bug that we need to dig into more: https://travis-ci.com/github/Shopify/ghostferry/builds/160296941